molecule/Dockerfile-centos8-base.j2

```diff
@@ -0,0 +1,41 @@
+FROM {{ item.image }}
```

Comment:
why is this file needed? Is it not possible to use one of the already existing docker files?

Reply:
This new Dockerfile is needed as we need to enable powertools repo using this file. Other Dockerfiles dont do that.